tests/framework/extensions/clusters/clusters.go

```diff
@@ -274,8 +275,12 @@ func NewK3SRKE2ClusterConfig(clusterName, namespace string, clustersConfig *Clus
 		SnapshotRetention:    5,
 		SnapshotScheduleCron: "0 */5 * * *",
 	}
-	if clustersConfig.Etcd != nil {
-		etcd = clustersConfig.Etcd
```

Comment:
Why wouldn't this work? Also where are you updating for rke1?

Reply:
RKE1 is not part of this PR, that was going to come in a separate PR.

Reply:
With the changes made in the PR, each field needs to be specified to work with the user's defined config file.